addon/templates/components/ui-button--default.hbs

```diff
@@ -4,19 +4,19 @@
   class="{{classes.parent}}
          {{classes.size}}
          {{classes.class}}
-         ui-button--default
-         {{if states.active 'ui-button--default--active'}}
-         {{if states.focus 'ui-button--default--focus'}}
-         {{if states.disabled 'ui-button--default--disabled'}}
-         {{if states.loading 'ui-button--default--loading'}}
-         {{if group 'ui-button--default--group-button'}}
-         {{if group.isFirstChild 'ui-button--default--first-child'}}
-         {{if group.isLastChild 'ui-button--default--last-child'}}">
-  <div class="ui-button--default__wrapper
-             {{if states.active 'ui-button--default__wrapper--active'}}">
-    <div class="ui-button--default__label
-               {{if states.loading 'ui-button--default__label--loading'}}
-               {{if states.active 'ui-button--default__label--active'}}">
+         :component
+         {{if states.active '--active'}}
+         {{if states.focus '--focus'}}
+         {{if states.disabled '--disabled'}}
+         {{if states.loading '--loading'}}
+         {{if group '--group-button'}}
+         {{if group.isFirstChild '--first-child'}}
+         {{if group.isLastChild '--last-child'}}">
+  <div class="__wrapper
+             {{if states.active '__wrapper--active'}}">
+    <div class="__label
+               {{if states.loading '__label--loading'}}
+               {{if states.active '__label--active'}}">
```

Comment:
i don't think we'll be able to know them. i've been considering just double dashing everything because of that.